packages/augur-sdk/src/Augur.ts

```diff
@@ -101,4 +101,16 @@ export class Augur<TProvider extends Provider = Provider> {
   public getOrders():ContractInterfaces.Orders {
     return new ContractInterfaces.Orders(this.dependencies, this.addresses.Orders);
   }
+
+  public registerTransactionStatusCallback(key: string, callback: TransactionStatusCallback): void {
```

Comment:
at the SDK level couldn't `key` be typed? is it `event name` or transaction type?

Reply:
Its arbitrary. its the name of the generic tx status update handler as provided by the consumer

Reply:
I see in the test "test" is used what would be an example the UI or other consumer would use?

Reply:
The key exists to give the consumer the ability to deregister the callback. I suppose we could return a closure the  does the same thing but that is more stuff to manage.

Reply:
Yea. In practice it would be something like "Notifications" for a handler specific to notifications.

Reply:
gotcha, that makes sense. so we'd only have one handler to manage tx in UI.

Reply:
If the same key is used twice that would kill the previous callback. Is that what we want here?

Reply:
I think that's fine personally. Alternative is to throw an error which I think is obnoxious.